kivy_garden/mapview/view.py

```diff
@@ -273,6 +273,24 @@ def unload(self):
         del self.markers[:]
 
 
+class PolylineLayer(MapLayer):
+    def __init__(self, coordinates, **kwargs):
+        super().__init__(**kwargs)
+        self.coordinates = coordinates
+
+    def reposition(self):
+        mapview = self.parent
+        with self.canvas:
+            self.canvas.clear()
+            Color(1, 0, 0, 1)  # Red color for the polyline
+            points = []
+            for lat, lon in self.coordinates:
+                x, y = mapview.get_window_xy_from(lat, lon, mapview.zoom)
+                points.extend([x, y])
+            if points:
+                SmoothLine(points=points, width=2, joint='round', cap='round')
```

Comment:
It would be better to cache the SmoothLine and just update the `.points` attribute on `reposition()`.